packages/content/src/lib/render-task.service.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Injectable,
+  inject,
+  ɵPendingTasks as PendingTasks,
+} from '@angular/core';
+import { filter, first, timeout } from 'rxjs';
+
+@Injectable()
+export class RenderTaskService {
+  pendingTasks = inject(PendingTasks);
```

Comment:
Thanks, It's good to know about how to use `PendingTasks`! 

Nit: `#pendingTasks = inject(PendingTasks)`?

Reply:
Fixed, thanks!